root/search/components/TagResults.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../../context';
+import {l} from '../../static/scripts/common/i18n';
+import TagLink from '../../static/scripts/common/components/TagLink';
+import loopParity from '../../utility/loopParity';
+import type {ResultsPropsT} from '../types';
+
+import PaginatedSearchResults from './PaginatedSearchResults';
+import ResultsLayout from './ResultsLayout';
+
+function buildResult(result, index) {
+  const tag = result.entity;
+  const score = result.score;
+
+  return (
+    <tr className={loopParity(index)} data-score={score} key={tag}>
```

Comment:
Not sure what the purpose of key here is. It's usually ID, but the tags don't have an ID - they're strings. As such, I changed it to be the tag itself - let me know if I should just remove key completely instead.

Reply:
It is for React to identify an item in a list (of `tr`, `li`, `p`… actually any HTML tag).
See https://reactjs.org/docs/lists-and-keys.html#keys